.github/workflows/test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Install nightly toolchain
         uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly
+          toolchain: nightly-2023-10-24
```

Comment:
Pretty sure there's way more places you need to set this if you're going to set it at all. For instance, it should be kept in lockstep with nix too.

Reply:
yeah i am just trying to get this job working